modules/ROOT/pages/configure-acm.adoc

```diff
@@ -202,6 +202,8 @@ The *ACM Control Panel* is the interface your community administrators and opera
 
 == Communities Configuration [[communities]]
```

Comment:
Note: super-quick copyedits only, and only on changed text, due to deadline.

Reply:
Do we need to worry about this text showing up here in italic?

Reply:
Note also that these tasks follow incorrect bullet hierarchy, which should be numbers followed by lowercase alphabet followed by lowercase Roman...but we should not get down to that level.
This doc still needs some large restructuring to avoid complicated text hierarchies.

Reply:
The italic is just a GitHub rendering quirk that doesn't affect the site.

I agree, the page has overly complicated hierarchies that will need to be restructured.